utils/params.go

```diff
@@ -706,3 +720,30 @@ func readConfigFromTarget(client vcsclient.VcsClient, gitParamsFromEnv *Git) (co
 	}
 	return
 }
+
+// This function fetches a config profile if JF_CONFIG_PROFILE is provided.
+// If so - it verifies there is only a single module with a '.' path from root. If these conditions doesn't hold we return an error.
+func getConfigProfileIfExistsAndValid(jfrogServer *coreconfig.ServerDetails) (configProfile *services.ConfigProfile, err error) {
+	profileName := getTrimmedEnv(JfrogConfigProfileEnv)
+	if profileName != "" {
+		configProfile, err = xsc.GetConfigProfile(jfrogServer, profileName)
+		if err != nil {
+			return
+		}
+
+		// Currently, only a single Module that represents the entire project is supported
+		if len(configProfile.Modules) != 1 {
+			err = fmt.Errorf("more than one module was found '%s' profile. Frogbot currently supports only one module per config profile", configProfile.ProfileName)
+			return
+		}
+		if configProfile.Modules[0].PathFromRoot != "." {
+			err = fmt.Errorf("module '%s' in profile '%s' contains the following path from root: '%s'. Frogbot currently supports only a single module with a '.' path from root", configProfile.Modules[0].ModuleName, profileName, configProfile.Modules[0].PathFromRoot)
+			return
+		}
+		log.Info(fmt.Sprintf("Config profile '%s' was found. All scanners configurations will be featched from env vars and the config profile. jfrog-apps-config will be ignored if exists", profileName))
+		log.Info("Utilizing config profile is currently intended for Jfrog internal use ONLY. If used- the only supported feature is enablement/exclusion of Secrets and Sast scanners")
+	} else {
+		log.Info("No JF_CONFIG_PROFILE environment variable was provided. All configurations will be induced from Env vars and files")
+	}
+	return
```

Comment:
```suggestion
	profileName := getTrimmedEnv(JfrogConfigProfileEnv)
    if profileName == "" {
        log.Debug(fmt.Sprintf("No %s environment variable was provided. All configurations will be induced from Env vars and files"), JfrogConfigProfileEnv)
        return
    }
	if configProfile, err = xsc.GetConfigProfile(jfrogServer, profileName); err != nil {
		return
	}
    // Currently, only a single Module that represents the entire project is supported
	if len(configProfile.Modules) != 1 {
		err = fmt.Errorf("more than one module was found '%s' profile. Frogbot currently supports only one module per config profile", configProfile.ProfileName)
		return
	}
    if configProfile.Modules[0].PathFromRoot != "." {
		err = fmt.Errorf("module '%s' in profile '%s' contains the following path from root: '%s'. Frogbot currently supports only a single module with a '.' path from root", configProfile.Modules[0].ModuleName, profileName, configProfile.Modules[0].PathFromRoot)
		return
	}
    log.Info(fmt.Sprintf("Config profile '%s' was found. All scanners configurations will be featched from env vars and the config profile. jfrog-apps-config will be ignored if exists", profileName))
	log.Info("Utilizing config profile is currently intended for Jfrog internal use ONLY. If used, the only supported feature is enablement/exclusion of Secrets and Sast scanners")
	return
```
* Keep code flat if possible
* I think logs about this should be in Debug unless found and used.
* Make sure the last log content is format, clear and approved